ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/assertion/DirectionsQuestionnaireAssert.java

```diff
@@ -0,0 +1,163 @@
+package uk.gov.hmcts.cmc.ccd.assertion;
+
+import org.assertj.core.api.AbstractAssert;
+import uk.gov.hmcts.cmc.ccd.domain.directionsquestionnaire.CCDDirectionsQuestionnaire;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.DirectionsQuestionnaire;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.ExpertRequest;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.HearingLocation;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.RequireSupport;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.Witness;
+
+import java.util.Objects;
+import java.util.function.Consumer;
+
+import static uk.gov.hmcts.cmc.ccd.assertion.Assertions.assertThat;
+
+public class DirectionsQuestionnaireAssert
+    extends AbstractAssert<DirectionsQuestionnaireAssert, DirectionsQuestionnaire> {
+
+    public DirectionsQuestionnaireAssert(DirectionsQuestionnaire actual) {
+        super(actual, DirectionsQuestionnaireAssert.class);
+    }
+
+    public DirectionsQuestionnaireAssert isEqualTo(CCDDirectionsQuestionnaire ccdDirectionsQuestionnaire) {
+
+        isNotNull();
+
+        isEqualToHearingLocation(ccdDirectionsQuestionnaire, actual.getHearingLocation());
+
+        actual.getRequireSupport().ifPresent(isEqualToRequireSupport(ccdDirectionsQuestionnaire));
+
+        actual.getWitness().ifPresent(isEqualToWitness(ccdDirectionsQuestionnaire));
+
+        actual.getExpertRequest().ifPresent(isEqualToExpertRequest(ccdDirectionsQuestionnaire));
+
+        return this;
+    }
+
+    private Consumer<Witness> isEqualToWitness(CCDDirectionsQuestionnaire ccdDirectionsQuestionnaire) {
```

Comment:
Why have we moved away from using Asserts?

Reply:
We haven't moved away from assertions. We introduce assertion where we have the whole object comparable in CCD domain as well.